terminus/builders/osm_city_builder.py

```diff
@@ -37,35 +37,30 @@ def get_city(self):
         # Get origin of the map (we use the center)
         tree = ET.parse(self.osm_map)
         bounds = tree.find('bounds').attrib
-        self.bounds = {'minlat': float(bounds['minlat']),
-                       'maxlat': float(bounds['maxlat']),
-                       'minlon': float(bounds['minlon']),
-                       'maxlon': float(bounds['maxlon'])}
+        self.bounds = {'origin': LatLon(float(bounds['minlat']), float(bounds['minlon'])),
```

Comment:
The maximum width of a line check (counted in number of characters) is disabled in pep8, but you should try to stick to 80 chars. In this case you should format the code in the lines of:
```
self.bounds = {'origin': LatLon(float(bounds['minlat']), 
                                float(bounds['minlon'])),
```

One good thing is to have sublime show the 80-char limit as a line (see e.g. http://stackoverflow.com/questions/25900954/80-characters-right-margin-line-in-sublime-text-3)